app/players/elves/elf.py

```diff
@@ -0,0 +1,19 @@
+from app.players.player import Player
+from abc import ABC, abstractmethod
+
+
+class Elf(Player, ABC):
+    def __init__(self, nickname: str, musical_instrument: str):
+        super().__init__(nickname)
+        self._musical_instrument = musical_instrument
+
+    @abstractmethod
+    def get_rating(self):
+        pass
+
+    @abstractmethod
+    def player_info(self):
+        pass
```

Comment:
If you have an abstract class that inherits another abstract class you don't need to repeat abstract methods from parent.